pkgs/development/python-modules/pecan/default.nix

```diff
@@ -52,7 +61,8 @@ buildPythonPackage rec {
 
   meta = with lib; {
     description = "Pecan";
```

Comment:
Something like
```suggestion
    description = "WSGI object-dispatching web framework, designed to be lean and fast";
```
might be better. You decide what fits.

Reply:
sounds good, changed it!